frontend/src/app/_components/creator-page/creator-page.component.html

```diff
@@ -0,0 +1,21 @@
+<app-breadcrumb [lastCrumbText]="creator.name"></app-breadcrumb>
+<div class="row creator-page">
+    <div class="col-sm-4 col-6">
```

Comment:
Why have a col-6 and then a col-12? Why not just make them both col-12? Is there any reason for it not to be full width on mobile?

Reply:
good call.